src/components/Display.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export default function Display() {
+  const input = '0';
+  return (
+    <div className="display">
+      {input}
+    </div>
+  );
+}
```

Comment:
- [ ] The display component should accept the **result** of the calculation as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png))
- [ ] Validate your props.
- [ ] The **result** prop should be a String with default value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))